user/enterprise/build-images.md

```diff
@@ -93,7 +95,34 @@ After that, both Docker and travis-worker have to be restarted. For Docker, plea
 $ sudo restart docker
 ```
 
-For travis-worker, you can find the instructions [here](https://docs.travis-ci.com/user/enterprise/worker-cli-commands/#Stopping-and-Starting-the-Worker).
+__Ubuntu 16.04 as Host operating system__
+
+To have the Docker daemon listening to the private IP address, please open `/etc/docker/daemon.json`. In there, please configure `hosts` like this:
+
+```json
+{
+    //...
+    "hosts": [
+        "fd://",
+        "tcp://172.31.7.199:4243"
+    ]
+}
+```
+
+By default, the Docker service file has configured that the daemon is accessible via socket activation. If you'd restart Docker right now, it would complain about having hosts configured both in the service file and in `/etc/docker/daemon.json`. To not run into that situation, please run `sudo systemctl edit docker`. It opens an editor with a new file where you can override Docker default settings. Please add the following in there:
```

Comment:
> (what even is socket activation??)

Socket activation is a new feature systemd introduced. Rather than having each service on the machine booting up and running, it's off by default but if any other service wants to talk to it the request / data gets queued and the service started. 
macOS is doing that exclusively as far as I know and the systemd people adapted that concept. 

Good catch with changing `ExecStart` -- if that override wouldn't happen Docker would complain about configuring it in two ways at the same time (that daemon json file and systemd service file) which is a problem for it (this is intentionally designed that way).

^ I'll add that explanation to the text too